.github/workflows/gpu.yml

```diff
@@ -63,6 +63,7 @@ jobs:
           -DUMF_BUILD_LIBUMF_POOL_DISJOINT=ON
           -DUMF_BUILD_LIBUMF_POOL_JEMALLOC=ON
           -DUMF_BUILD_LEVEL_ZERO_PROVIDER=ON
+          -DUMF_BUILD_CUDA_PROVIDER=ON
```

Comment:
if you want to run this on CUDA runner, you can use `["DSS-CUDA", "DSS-UBUNTU"]` (only Ubuntu is available for now)